specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -176,52 +176,32 @@ export function IntegratedRecordSelector({
     relationship.relatedTable.name
   );
 
-  const [isCollapsed, _, handleCollapsed, handleToggle] =
+  const [isCollapsed, _handleCollapsed, handleExpand, handleToggle] =
```

Comment:
FYI in case you were wondering why I used `_handleCollapsed` instead of `_` here.

There is no difference in behavior.

There is a very common convention in JavaScript that any variable that begins with `_` means unused - even if it's just `_` or `__`. Thus, the only reason to do something like `_handleCollapsed` instead of `_` is that the longer format is more explicit about what the unused variable is for - this makes it less guesswork for anyone reading the code, and if that unused variable will become used at a later point, naming it like this makes it easier to find

Not a big deal and most of the time I still go with `_` for simplicity, but just wanted you to know about the reasons behind it